cmd/hyperconverged-cluster-operator/main.go

```diff
@@ -142,8 +155,13 @@ func main() {
 
 	ctx := context.TODO()
 
+	lockName := "hyperconverged-cluster-operator-lock"
+	if operatorWebhookMode {
```

Comment:
Is leader election actually necessary for webhook mode?

Reply:
You are right, it's not: OLM it's going to expose it over a service that can have more than one endpoint.
I'm going to fix it.